src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -94,4 +138,18 @@ private WebElement getDraftTab() {
     private WebElement getTrashTab() {
         return browser.driver.findElement(By.cssSelector("a[href='" + Const.ActionURIs.ADMIN_EMAIL_TRASH_PAGE + "']"));
     }
+
+    /**
+     * Verifies a substring in encoded key which remains same for a particular file type.
+     *
+     *  @param key in which a particular subString has to be checked
```

Comment:
Update `@param` or you can just remove this. The parameter is already self-plained.